### PR TITLE
fix: add zh readme

### PR DESCRIPTION
**Pull request by zhangqx2010**
Translated according to these 3 docs.

https://raw.githubusercontent.com/kubernetes/website/4bb98f739582d0653060e281061e2abdb78f1eaf/README.md
https://github.com/kubernetes/website/blob/master/README.md
https://github.com/kubernetes/website/blob/master/README-de.md

### CODE REVIEW

**Comment by zhangqx2010** [comment]
fix #14773

**Comment by zhangqx2010** [comment]
@tengqm Please take a look. Thanks.

**Comment by zhangqx2010** [comment]
cc @markthink

File: README-zh.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+欢迎！本仓库包含了所有用于构建 [Kubernetes 网站和文档](https://kubernetes.io/)的内容。
+我们非常高兴您想要参与贡献！
+
+## 贡献文档
+
+您可以点击屏幕右上方都 **Fork** 按钮，在您的 Github 账户下拷贝创建一份仓库副本。这个副本叫做 *fork*。您可以对 fork 进行任意修改，
```
Comment (tengqm) [inline README-zh.md]: > 都 .. 按钮
的 .. 按钮

这是一个笔误。不过，强烈建议你将英文原文以 HTML 注释的形式包含在此 Markdown 文件中以便追踪变更。

File: README-zh.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+创建拉取请求后，Kubernetes 审核人员将负责提供清晰且可操作的反馈。作为拉取请求的所有者，**您有责任修改拉取请求以解决 Kubernetes 审核者提供给您的反馈。**
+另请注意，您最终可能收到多个 Kubernetes 审核人员为您提供反馈，或者您可能会收到 Kubernetes 审核人员最终的反馈与最初的反馈不尽相同。
+此外，在某些情况下，您的一位评审员可能会在需要时要求 [Kubernetes 技术评审员](https://github.com/kubernetes/website/wiki/Tech-reviewers) 进行技术评审。
+审稿人将尽最大努力及时提供反馈，但响应时间可能因情况而异。
+
+有关为 Kubernetes 文档做出贡献的更多信息，请参阅：
+
+* [开始贡献](https://kubernetes.io/docs/contribute/start/)
+* [换成您的文档变更](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
```
Comment (tengqm) [inline README-zh.md]: 这个译文好像不对

**Comment by tengqm** [comment]
Looks good in general. Please include the original text in your localized version for change management and review. Thanks.

File: README-zh.md (hunk 1 above)
Comment (zhangqx2010) [reply README-zh.md]: 好的，我调整一下。原始英文的文档格式是直接调整的，我根据三个文档拼接了一下。

**Comment by zhangqx2010** [comment]
#14562

**Comment by zhangqx2010** [comment]
@tengqm @chenrui333 Please take a look. Thanks.

File: README-zh.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+* [开始贡献](https://kubernetes.io/docs/contribute/start/)
+* [缓存您的文档变更](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [使用页面模版](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [文档风格指南](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [本地化 Kubernetes 文档](https://kubernetes.io/docs/contribute/localization/)
+
+<!--
+## `README.md`'s Localizing Kubernetes Documentation
+ -->
+## `README.md` 的本地化 Kubernetes 文档
```
[the remark this answers is not in the document]
Comment (zhangqx2010) [reply README-zh.md]: 这个我还是保留吧，与所有文档保持一致

File: README-zh.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+当准备好把变更提交给我们时，您可以通过创建一个 pull request 来通知我们。
+
+<!--
+Once your pull request is created, a Kubernetes reviewer will take responsibility for providing clear, actionable feedback.
+As the owner of the pull request, **it is your responsibility to modify your pull request to address the feedback that has been provided to you by the Kubernetes reviewer.**
+Also, note that you may end up having more than one Kubernetes reviewer provide you feedback or you may end up getting feedback from a Kubernetes reviewer that is different than the one initially assigned to provide you feedback.
+Furthermore, in some cases, one of your reviewers might ask for a technical review from a [Kubernetes tech reviewer](https://github.com/kubernetes/website/wiki/Tech-reviewers) when needed.
+Reviewers will do their best to provide feedback in a timely fashion but response time can vary based on circumstances.
+ -->
+创建拉取请求后，Kubernetes 审核人员将负责提供清晰且可操作的反馈。作为拉取请求的所有者，**您有责任修改拉取请求以解决 Kubernetes 审核者提供给您的反馈。**
```
Comment (SataQiu) [inline README-zh.md]:
```suggestion
创建 pull request 后，Kubernetes 审核人员将负责提供清晰且可操作的反馈。作为 pull request 的所有者，**您有责任修改 pull request 以解决 Kubernetes 审核者提供给您的反馈。**
```

Comment (SataQiu) [reply README-zh.md]: 这里要么就也改了吧，前后一致

Comment (zhangqx2010) [reply README-zh.md]: 多谢review 👍 😄

**Comment by SataQiu** [comment]
Thanks for your update @zhangqx2010 ! :blush:
/lgtm

**Comment by chenrui333** [comment]
I will take a look later, nice work at first!!

File: README-zh.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+ -->
+## 贡献文档
+
+<!--
+You can click the **Fork** button in the upper-right area of the screen to create a copy of this repository in your GitHub account.
+This copy is called a *fork*. Make any changes you want in your fork,
+and when you are ready to send those changes to us, go to your fork and create a new pull request to let us know about it.
+ -->
+您可以点击屏幕右上方的 **Fork** 按钮，在您的 GitHub 账户下创建一份本仓库的副本。这个副本叫做 *fork*。您可以对 fork 副本进行任意修改，
+当准备好把变更提交给我们时，您可以通过创建一个 pull request 来通知我们。
```
Comment (chenrui333) [inline README-zh.md]:
```suggestion
当准备好把修改提交给我们时，您可以通过创建一个 pull request 来告知我们。
```

File: README-zh.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+
+* June Yi ([GitHub - @gochist](https://github.com/gochist))
+* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-ko)
+ -->
+### 中文
+
+可以通过以下方式联系中文本地化的维护人员：
+
+* He Xiaolong ([GitHub - @markthink](https://github.com/markthink))
+* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-zh)
```
Comment (chenrui333) [inline README-zh.md]: you can also add my info in here too. 

`Rui Chen ([GitHub - @chenrui333](https://github.com/chenrui333))`

Comment (zhangqx2010) [reply README-zh.md]: Of course, I did think of that. 😄

**Comment by zhangqx2010** [comment]
need lgtm @chenrui333

**Comment by chenrui333** [comment]
/lgtm 
/approve

Thanks!!

File: README-zh.md (hunk 6 above)
Comment (chenrui333) [reply README-zh.md]: haha 👍

**Comment by chenrui333** [comment]
@kbarnard10, It is ready to ship!

**Pull request merged**